apps/site/pages/components/molecules/modal.mdx

```diff
@@ -0,0 +1,285 @@
+---
+description: Modals are dialog windows that sit on top of an application's main view.
+sidebar_custom_props:
+  image: https://vtexhelp.vtexassets.com/assets/docs/src/Modal___16993b8e327f3571529d7ee4e97cb45b.png
+---
+
+import { Tab, Tabs } from 'nextra-theme-docs'
+import PropsSection from 'site/components/PropsSection'
+import { TokenTable, TokenRow, TokenDivider } from 'site/components/Tokens'
+import { OverviewSection } from 'site/components/OverviewSection'
+import {
+  Modal,
+  ModalHeader,
+  ModalBody,
+  useUI,
+  UIProvider,
+  Button,
+} from '@faststore/ui'
+
+<header>
+
+# Modal
+
+Modals are dialog windows that sit on top of an application's main view.
+
+</header>
+
+- `Modal`: Dialog windows with card style base.
+- `ModalHeader`: Optional header component for Modal, with title, description, and close button.
+- `ModalBody`: Optional Body component wrapper to contents for modal.
```

Comment:
```suggestion
- `ModalHeader`: Wraps the Modal header. It may contain a title, description and a close button. (Optional)
- `ModalBody`: Wraps the Modal body content. (Optional)
```